app/controllers/api/v1/cars_controller.rb

```diff
@@ -0,0 +1,62 @@
+class Api::V1::CarsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  # /api/v1/cars
+  def index
+    @cars = Car.all
+    render json: all_cars_with_image(@cars)
```

Comment:
- [ ] This is valid for the `car_object` method too, but aren't `@cars` and `@car` instance variables? And if they are, then you should be able to access them without passing them as parameters to the method. If this is not working somehow, then I suggest you try refactoring what you are trying to achieve here, because it doesn't look very clean in my honest opinion.

Reply:
@MuriloRoque we are using the car_object method to recreate the object in a way that the image is no longer represented as an active storage blob but rather as "image: Cloudinary::Utils.cloudinary_url(car.image.key)" since we're using cloudinary to store our images we'd want to have it in front end as api_request_data.image to get the link of the image from cloudinary